src/cli/test/test_cli.c

```diff
@@ -1180,6 +1230,31 @@ static void test_trn_cli_update_net_subcmd(void **state)
 	expect_any(__wrap_update_net_1, clnt);
 	rc = trn_cli_update_net_subcmd(NULL, argc, argv1);
 	assert_int_equal(rc, -EINVAL);
+
```

Comment:
side note: these test cases seem to have a lot of repetitions.

consider putting these in a loop.

Reply:
I agree, this is starting to get bloated. Please make this table driven. See example: https://github.com/CentaurusInfra/mizar/blob/dev-next/src/cli/test/test_cli.c#L1945

Reply:
Updated